kimchi/src/circuits/polynomials/range_check/circuitgates.rs

```diff
@@ -202,6 +202,15 @@ where
         // Check value v against the sum of limbs
         constraints.push(sum_of_limbs - env.witness_curr(0));
 
+        // Optional compact limbs format (enabled when coeff[0] == 1, disabled when coeff[1] = 0)
+        //   Constrain decomposition of compact limb next(1)
+        //   next(1) = curr(0) + 2^L * next(0)
+        constraints.push(
+            env.coeff(0)
+                * (env.witness_next(1)
+                    - (env.witness_curr(0) + T::two_to_limb() * env.witness_next(0))),
```

Comment:
It's v01, but I just map it to cells v12 but again, the names of the range-check cells are arbitrary. That is, how it's used v12 is v01.

Reply:
I didn't want to make this PR too huge and rename everything.  I feel like that is another PR because it impacts normal mode of operation and is a different goal.